packages/runtime/container-runtime/src/deltaScheduler.ts

```diff
@@ -86,7 +90,18 @@ export class DeltaScheduler {
 
                 // eslint-disable-next-line @typescript-eslint/no-floating-promises
                 this.deltaManager.inbound.pause();
+
                 setTimeout(() => {
+                    if (this.schedulingLog) {
```

Comment:
Why is this inside setTimeout? Shouldn't this be logged before we call setTimeout?

Reply:
The idea was to be able to get info of when the resume happens....

Reply:
I see. `waitingToResume` is also something we want to log so that makes sense :)